### DISCUSSION TITLE
[Feature] A button to go from a photo shown in an album to the photo in all photos

### DISCUSSION THREAD

**Discussion by coryrearick9900** [immich-app/immich#12761] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

On the iOS photos app you can press a button below a photo that brings you to that photo in all the photos you have.  I think this would be a useful feature in Immich.

![image](https://github.com/user-attachments/assets/152b2582-3641-46d1-bf82-0c07e8c2436f)

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
You can do this from the drop down menu in the asset view 
![image](https://github.com/user-attachments/assets/2e2b063b-82fb-4a47-a1b2-6a4d6027aa63)

**Reply by dmitchell78721** [reply]
On the web yes, but not in the iPhone app as far as I can tell.
